docker-infrastructure.yml

```diff
@@ -34,19 +34,17 @@ services:
     # default 'testclientid' client id
     command: >-
       --client_id=${ACRA_CLIENT_ID:-poison_records_demo}
-      --generate_acraserver_keys
+      --generate_symmetric_storage_key
       --generate_acrawriter_keys
```

Comment:
do we need to generate AW key?

Reply:
I guess yes, because we use transparent encryption in this example :eyes:

Reply:
`generate_acrawriter_keys` these are asymmetric keys, why we need them, if we are generating symmetric keys now? cc @Lagovas

Reply:
Just checked - Acra fails to insert poison record with `level=error msg="Key found error" client_id=poison_records_demo code=1100 error="can't read private key for matched client_id[/zone_id]() to decrypt AcraStruct` without these keys.
I guess that's because Acra tries to decrypt the poison record, before insertion. But poison record is generated as AcraStruct by default instead of AcraBlock.

Reply:
Yes, we can. We should explicitly set type of poison record - https://github.com/cossacklabs/acra/blob/master/cmd/acra-poisonrecordmaker/acra-poisonrecordmaker.go#L64
And if I'm correct, we will not see `can't read private key for matched client_id[/zone_id]() to decrypt AcraStruct` because it will not try to decrypt acrastruct (because will not find it)